frontend/online-shop/src/styles/ProductListStyle.tsx

```diff
@@ -0,0 +1,30 @@
+import { makeStyles, createStyles, Theme } from "@material-ui/core";
+
+export const productListStyle = makeStyles((theme: Theme) =>
+  createStyles({
+    tableStyle: {
+      width: "750px",
+      border: "2px solid",
+      borderCollapse: "collapse",
+      marginLeft: "40px",
+    },
+
```

Comment:
Again, no need for spacing between style classes either

Reply:
Fixed.